src/components/views/elements/CustomInput.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useRef, useEffect } from 'react';
+
+// via https://itnext.io/reusing-the-ref-from-forwardref-with-react-hooks-4ce9df693dd
+function useCombinedRefs(...refs) {
```

Comment:
ideally this gets put with some of the other generic hooks instead of buried in a file we'll forget about :)